frontend/app/dtos/parse-address-response-dto.server.ts

```diff
@@ -0,0 +1,120 @@
+export interface ParseWSAddressResponseDTOProps {
+  responseType?: string;
+  addressLine?: string;
+  city?: string;
+  province?: string;
+  postalCode?: string;
+  streetNumberSuffix?: string;
+  streetDirection?: string;
+  unitType?: string;
+  unitNumber?: string;
+  serviceAreaName?: string;
+  serviceAreaType?: string;
+  serviceAreaQualifier?: string;
+  cityLong?: string;
+  cityShort?: string;
+  deliveryInformation?: string;
+  extraInformation?: string;
+  statusCode?: string;
+  canadaPostInformation?: string[];
+  message?: string;
+  addressType?: string;
+  streetNumber?: string;
+  streetName?: string;
+  streetType?: string;
+  serviceType?: string;
+  serviceNumber?: string;
+  country?: string;
+  warnings?: string;
+  functionalMessages?: {action: string, message: string}[];
+}
```

Comment:
See other comment about moving this into the `wsaddress-service.server.ts` file

Reply:
Moved response DTO to wsaddress-service